### PR TITLE
enhancement(Updates): use buttons for admin actions

### PR DESCRIPTION
**Pull request by saideepesh000**
Resolve [opencollective-frontend/issues/3879](https://github.com/opencollective/opencollective/issues/3879)

# Description

 This PR Fixes the styling of the edit/delete buttons that are on the [collective]/Updates/[updates] page

# Screenshots
Screenshot (6)

### CODE REVIEW

**Comment by saideepesh000** [comment]
Should I completely remove the  "ActionButton" Styled Component

File: components/StyledUpdate.js
Comment (Betree) [inline components/StyledUpdate.js:195]: While we're at it, we should use the same capitalization than "Edit". After merging this change, you'll need to run `npm run build:langs` to update the language files and commit the result.

```suggestion
                <FormattedMessage id="update.delete" defaultMessage="Delete" />
```

**Review by Betree — COMMENTED** [review]
It's looking great!

> Should I completely remove the "ActionButton" Styled Component

Yes, as you can notice on CI's results it won't let you merge otherwise: https://github.com/opencollective/opencollective-frontend/pull/5793/checks?check_run_id=1777986828

**Comment by saideepesh000** [comment]
Hey @Betree, it seems like you assigned it yourself, should I continue to make changes or start working on some other issues.

**Comment by SudharakaP** [comment]
> Hey @Betree, it seems like you assigned it yourself, should I continue to make changes or start working on some other issues.

@saideepesh000 : Maybe Benjamin can chime in; but the assigning is for his own tracking; if you notice Benjamin assigns himself every PR that he reviews. Therefore you should continue working on it. 😄 

(@Betree : Sorry for commenting on your behalf; thought it would be useful given the timezone difference so that @saideepesh000 can start with the changes).

**Comment by Betree** [comment]
> > Hey @Betree, it seems like you assigned it yourself, should I continue to make changes or start working on some other issues.
> 
> @saideepesh000 : Maybe Benjamin can chime in; but the assigning is for his own tracking; if you notice Benjamin assigns himself every PR that he reviews. Therefore you should continue working on it. smile
> 
> (@Betree : Sorry for commenting on your behalf; thought it would be useful given the timezone difference so that @saideepesh000 can start with the changes).

It's ok! Your explanation is correct, all contributor's PRs must be assigned to someone from the core team that is responsible for reviewing/merging it and followup if needed.

**Review by Betree — APPROVED** [review]
This is looking great, nice job @saideepesh000!

**Review by snitin315 — APPROVED** [review]
💯

**Pull request merged**
